data/data-pipeline/data_pipeline/utils.py

```diff
@@ -198,6 +199,21 @@ def temp_folder_cleanup() -> None:
     remove_all_from_dir(data_path / "tmp")
 
 
+def check_first_run() -> bool:
+    """Checks if a local semaphore has been set to only run once the
+    calling function"""
+
+    data_path = settings.APP_ROOT / "data"
+    file = "first_run.txt"
+
+    if not os.path.isfile(data_path / file):
+        cmd = f"touch {data_path}/{file}"
+        call(cmd, shell=True)
```

Comment:
does this file ever get deleted?

Reply:
Per the description I just added to the PR above, no, it has to be manually deleted by the user, but it's not checked in to the repo since it's on a git ignored directory and only influences the application if the command is called with the `-c` flag.